proposals/2018/Rename Root Element.md

```diff
@@ -0,0 +1,53 @@
+# Rename the Root Element
```

Comment:
this file is here due to an unnecessary merge. The file is not part of this proposal but will not cause any issues.